hazelcast-spring-tests/src/test/resources/com/hazelcast/spring/fullConfig-applicationContext-hazelcast.xml

```diff
@@ -1061,6 +1061,10 @@
                     <hz:receive-window-multiplier>4</hz:receive-window-multiplier>
                 </hz:edge-defaults>
             </hz:jet>
+            <hz:device>
+                <hz:device-name>tiered_store_device</hz:device-name>
```

Comment:
We also should add read-io-threads-count and write-io-threads-count

Reply:
No, we don't reference device config from `tieredStoreConfig` or `mapConfig`, since device config is more global and local to a given HZ instance. 

Ok, thanks, I will add read/write io thread counts